extension-manifest-v3/src/pages/offscreen/urlfilter2dnr/index.js

```diff
@@ -0,0 +1,9 @@
+import convert from '/utils/dnr-converter.js';
+
+chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
+  if (msg.action === 'offscreen:urlfitler2dnr:convert') {
+    convert(msg.filter).then((result) => sendResponse(result));
```

Comment:
What if "convert" throws?

(If there is nothing better, then adding a ".catch(console.error)" could be useful. Even if convert should never fail.)

Reply:
it never throws, returns `errors` instead